python/hail/context.py

```diff
@@ -55,10 +55,10 @@ class HailContext(HistoryMixin):
                       append=bool,
                       min_block_size=integral,
                       branching_factor=integral,
-                      tmp_dir=strlike)
+                      maybe_tmp_dir=nullable(strlike))
```

Comment:
I'd prefer leaving it as tmp_dir, since if it's passed it overrides the env. We don't use optional indicators like 'maybe' much elsewhere, so it seems a bit out of place here. 

Feel free to push back.

Reply:
I forgot I can mutate arguments python 👍